src/components/dialogs/reactive-limits/reactive-capability-curve/reactive-capability-curve-table.jsx

```diff
@@ -45,6 +46,22 @@ export const ReactiveCapabilityCurveTable = ({
         remove(index);
     };
 
+    useEffect(() => {
+        if (rows?.length < MIN_LENGTH) {
+            for (let i = 0; i < MIN_LENGTH - rows.length; i++) {
+                if (previousValues && updatePreviousReactiveCapabilityCurveTable) {
+                    updatePreviousReactiveCapabilityCurveTable(INSERT, rows.length);
+                }
+
+                insert(rows.length, {
+                    [P]: null,
+                    [MIN_Q]: null,
+                    [MAX_Q]: null,
+                });
+            }
+        }
```

Comment:
This is now a copy paste of `handleInsertRow`. Surprisingly sonar doesn't complain, but I think this should be refactored into a function.